libs/ui/src/components/token/account-token/account-token.tsx

```diff
@@ -1,79 +1,49 @@
-import { isDefined, isEmptyString } from '@rnw-community/shared';
+import { isDefined } from '@rnw-community/shared';
+import { BigNumber } from 'bignumber.js';
 import React, { FC } from 'react';
 import { Pressable } from 'react-native';
 import { useDispatch } from 'react-redux';
 
-import { DATA_UPDATE_TIME } from '../../../constants/update-time';
 import { ScreensEnum } from '../../../enums/sreens.enum';
 import { useNavigation } from '../../../hooks/use-navigation.hook';
-import { useTimerEffect } from '../../../hooks/use-timer-effect.hook';
-import { Token } from '../../../interfaces/token.interface';
+import { TokenWithFiatBalance } from '../../../interfaces/token.interface';
 import { getImageSource } from '../../../screens/wallet/components/assets-widget/utils/get-image-source.util';
-import { TokenPriceInfo } from '../../../store/tokens-market-info/tokens-market-info.state';
-import {
-  changeTokenVisibilityAction,
-  loadAccountTokenBalanceAction,
-  loadGasTokenBalanceAction
-} from '../../../store/wallet/wallet.actions';
-import {
-  useSelectedAccountPublicKeyHashSelector,
-  useSelectedNetworkSelector
-} from '../../../store/wallet/wallet.selectors';
+import { changeTokenVisibilityAction } from '../../../store/wallet/wallet.actions';
 import { checkIsGasToken } from '../../../utils/check-is-gas-token.util';
-import { getDollarValue } from '../../../utils/get-dollar-amount.util';
-import { getFormattedBalance } from '../../../utils/units.utils';
+import { getFiatBalanceToDisplay } from '../../../utils/get-dollar-value-to-display.util';
 import { Switch } from '../../switch/switch';
 import { TokenItemThemesEnum } from '../token-item/enums';
 import { TokenItem } from '../token-item/token-item';
 
 interface Props {
-  token: Token;
-  marketInfo?: TokenPriceInfo;
+  token: TokenWithFiatBalance;
   showButton?: boolean;
-  loadBalance?: boolean;
   theme?: TokenItemThemesEnum;
 }
 
-export const AccountToken: FC<Props> = ({ token, showButton, loadBalance = false, theme, marketInfo = {} }) => {
+export const AccountToken: FC<Props> = ({ token, showButton, theme }) => {
   const dispatch = useDispatch();
   const { navigate } = useNavigation();
-  const { rpcUrl } = useSelectedNetworkSelector();
-  const publicKeyHash = useSelectedAccountPublicKeyHashSelector();
-  const { price } = marketInfo;
-  const { decimals, thumbnailUri, balance, symbol, name, tokenAddress } = token;
+  const { thumbnailUri, balance, symbol, name, tokenAddress, fiatBalance } = token;
   const isGasToken = checkIsGasToken(tokenAddress);
 
+  const fiatBalanceToDispaly = getFiatBalanceToDisplay(new BigNumber(token.balance.data), fiatBalance);
```

Comment:
could you make  `fiatBalance` do be formatted like `getFiatBalanceToDisplay` in the place where `fiatBalance` is defined?